app/views/name/edit.html.erb

```diff
@@ -0,0 +1,30 @@
+<% content_for :page_title, "#{'Error: ' if @name_form.errors.any?}Your name" %>
+<% content_for :back_link_url, back_link_url(@name_form) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      Your name
+    </h1>
+    <p class="govuk-body">We’ll match your name against the Teaching Regulation Agency records.</p>
+    <%= form_with model: @name_form, url: name_path, method: :patch do |f| %>
+      <%= f.govuk_error_summary %>
+      <%= f.govuk_text_field(:first_name,
+            label: { size: 'm', text: 'First name' },
+            required: true
+          ) %>
+      <%= f.govuk_text_field(:last_name,
+            label: { size: 'm', text: 'Last name' },
+            required: true
+          ) %>
+
+      <%= f.govuk_check_boxes_fieldset :name_changed, legend: nil do %>
```

Comment:
Found a small bug locally; when clicking the `Change name` link on the check answers page when I've entered a previous name, the checkbox renders unticked, so the previous names are hidden on initial page load.

Reply:
Fixed. 👍